dlpi/src/sys.rs

```diff
@@ -1,6 +1,6 @@
-use num_enum::TryFromPrimitive;
```

Comment:
Should this file just go away in favor of libdlpi-sys/src/lib?

Reply:
Almost certainly!  I think I forgot to delete it basically.  I'll give that a swing.